tailwind.config.cjs

```diff
@@ -1,8 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
   mode: 'jit', // update this line
-  purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
-  darkMode: false,
```

Comment:
What did this do?

Reply:
Obsolete with Tailwind 3.0. The console kept warning about it.